eng/Versions.props

```diff
@@ -19,6 +19,12 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
   </PropertyGroup>
+  <PropertyGroup>
+    <VersionFeature21>30</VersionFeature21>
+    <VersionFeature31>32</VersionFeature31>
+    <VersionFeature50>17</VersionFeature50>
+    <VersionFeature60>$([MSBuild]::Add($(VersionFeature), 11))</VersionFeature60>
```

Comment:
@marcpopMSFT should we hold off on this until https://github.com/dotnet/installer/pull/15776 is in, or take this instead? This needs to change from 11 to 13, but other than that this should be good to go

Reply:
didn't see this before merging the other one so...uh yes.